data/script_cmd_table.inc

```diff
@@ -1,52 +1,54 @@
+	.4byte MUT_INSTRUMENTED(ScrCmd_call_if)         @ 0x07
+	.4byte MUT_INSTRUMENTED(ScrCmd_gotostd)         @ 0x08
+	.4byte MUT_INSTRUMENTED(ScrCmd_callstd)         @ 0x09
+	.4byte MUT_INSTRUMENTED(ScrCmd_gotostd_if)      @ 0x0a
+	.4byte MUT_INSTRUMENTED(ScrCmd_callstd_if)      @ 0x0b
 	.4byte ScrCmd_returnram                         @ 0x0c
 	.4byte ScrCmd_endram                            @ 0x0d
 	.4byte ScrCmd_setmysteryeventstatus             @ 0x0e
-	.4byte ScrCmd_loadword                          @ 0x0f
-	.4byte ScrCmd_loadbyte                          @ 0x10
+	.4byte MUT_INSTRUMENTED(ScrCmd_loadword)        @ 0x0f
+	.4byte MUT_INSTRUMENTED(ScrCmd_loadbyte)        @ 0x10
 	.4byte ScrCmd_setptr                            @ 0x11
-	.4byte ScrCmd_loadbytefromptr                   @ 0x12
+	.4byte MUT_INSTRUMENTED(ScrCmd_loadbytefromptr) @ 0x12
 	.4byte ScrCmd_setptrbyte                        @ 0x13
-	.4byte ScrCmd_copylocal                         @ 0x14
+	.4byte MUT_INSTRUMENTED(ScrCmd_copylocal)       @ 0x14
 	.4byte ScrCmd_copybyte                          @ 0x15
-	.4byte ScrCmd_setvar                            @ 0x16
-	.4byte ScrCmd_addvar                            @ 0x17
-	.4byte ScrCmd_subvar                            @ 0x18
-	.4byte ScrCmd_copyvar                           @ 0x19
-	.4byte ScrCmd_setorcopyvar                      @ 0x1a
-	.4byte ScrCmd_compare_local_to_local            @ 0x1b
-	.4byte ScrCmd_compare_local_to_value            @ 0x1c
-	.4byte ScrCmd_compare_local_to_ptr              @ 0x1d
-	.4byte ScrCmd_compare_ptr_to_local              @ 0x1e
-	.4byte ScrCmd_compare_ptr_to_value              @ 0x1f
-	.4byte ScrCmd_compare_ptr_to_ptr                @ 0x20
-	.4byte ScrCmd_compare_var_to_value              @ 0x21
-	.4byte ScrCmd_compare_var_to_var                @ 0x22
-	.4byte ScrCmd_callnative                        @ 0x23
-	.4byte ScrCmd_gotonative                        @ 0x24
-	.4byte ScrCmd_special                           @ 0x25
-	.4byte ScrCmd_specialvar                        @ 0x26
+	.4byte MUT_INSTRUMENTED(ScrCmd_setvar)          @ 0x16
+	.4byte MUT_INSTRUMENTED(ScrCmd_addvar)          @ 0x17
+	.4byte MUT_INSTRUMENTED(ScrCmd_subvar)          @ 0x18
+	.4byte MUT_INSTRUMENTED(ScrCmd_copyvar)         @ 0x19
+	.4byte MUT_INSTRUMENTED(ScrCmd_setorcopyvar)    @ 0x1a
+	.4byte MUT_INSTRUMENTED(ScrCmd_compare_local_to_local) @ 0x1b
+	.4byte MUT_INSTRUMENTED(ScrCmd_compare_local_to_value) @ 0x1c
+	.4byte MUT_INSTRUMENTED(ScrCmd_compare_local_to_ptr) @ 0x1d
+	.4byte MUT_INSTRUMENTED(ScrCmd_compare_ptr_to_local) @ 0x1e
+	.4byte MUT_INSTRUMENTED(ScrCmd_compare_ptr_to_value) @ 0x1f
+	.4byte MUT_INSTRUMENTED(ScrCmd_compare_ptr_to_ptr) @ 0x20
+	.4byte MUT_INSTRUMENTED(ScrCmd_compare_var_to_value) @ 0x21
+	.4byte MUT_INSTRUMENTED(ScrCmd_compare_var_to_var) @ 0x22
+	.4byte MUT_INSTRUMENTED(ScrCmd_callnative)      @ 0x23
+	.4byte MUT_INSTRUMENTED(ScrCmd_gotonative)      @ 0x24
```

Comment:
All comments need realignment. I wonder if we can make this list less prone to breaking by moving it into .c, much like how we made the battle effects easier to edit. This should also make it easier for people to avoid custom script commands being out of order. Possibly out of scope, though.

Reply:
I'd deliberately not adjusted the comments to avoid causing conflicts for any downstream projects that added their own commands in places that haven't been touched by this PR.

I can align them if we think that's better? It's certainly not very aesthetically-pleasing atm in the PR.

Reply:
> This should also make it easier for people to avoid custom script commands being out of order. 

Regarding this, there's [pret#2005](https://github.com/pret/pokeemerald/pull/2005) which attempts to tackle the ordering problem from a different direction.

Reply:
Ah, I didn't see the pret PR yet. Would be nice if it could get merged in there at some point I suppose :)

As for the alignment, I believe we have always fixed alignment in expansion. It should be trivial to fix that merge conflict (I think letting VSCode combine changes does like 90% of the work, even).